deployment/docker/rules/alerts-cluster.yml

```diff
@@ -11,13 +11,12 @@ groups:
     rules:
       - alert: DiskRunsOutOfSpaceIn3Days
         expr: |
-          vm_free_disk_space_bytes / ignoring(path)
+          sum(vm_free_disk_space_bytes) without(path) /
           (
-            rate(vm_rows_added_to_storage_total[1d])
-            * scalar(
-              sum(vm_data_size_bytes{type!~"indexdb.*"}) /
-              sum(vm_rows{type!~"indexdb.*"})
-             )
+            rate(vm_rows_added_to_storage_total[1d]) * (
```

Comment:
I haven't tried it. why do you think it's not compatible?

Reply:
>I haven't tried it. why do you think it's not compatible?

I assumed so because `scalar` likely was added for a reason.

Try the following [similar query](http://demo.robustperception.io:9090/query?g0.expr=rate%28go_gc_pauses_seconds_count%5B5m%5D%29+*+scalar%28sum%28go_gc_heap_goal_bytes%29+%2F+sum%28go_gc_heap_live_bytes%29%29&g0.show_tree=0&g0.tab=table&g0.range_input=1h&g0.res_type=auto&g0.res_density=medium&g0.display_mode=lines&g0.show_exemplars=0).

Reply:
I've updated it to [one](http://demo.robustperception.io:9090/query?g0.expr=rate%28go_gc_pauses_seconds_count%5B5m%5D%29+*+%28sum%28go_gc_heap_goal_bytes%29+by+%28instance%2C+job%29+%2F+sum%28go_gc_heap_live_bytes%29+by+%28instance%2C+job%29%29&g0.show_tree=0&g0.tab=table&g0.range_input=1h&g0.res_type=auto&g0.res_density=medium&g0.display_mode=lines&g0.show_exemplars=0), which also works

Reply:
That's weird, I'd expect both queries to fail [here](http://demo.robustperception.io:9090/query?g0.expr=rate%28go_gc_pauses_seconds_count%5B5m%5D%29+*+%28sum%28go_gc_heap_goal_bytes%29+without%28lol%29+%2F+sum%28go_gc_heap_live_bytes%29+without%28lol%29%29&g0.show_tree=0&g0.tab=table&g0.range_input=1h&g0.res_type=auto&g0.res_density=medium&g0.display_mode=lines&g0.show_exemplars=0&g1.expr=rate%28go_gc_pauses_seconds_count%5B5m%5D%29+*+%28sum%28go_gc_heap_goal_bytes%29+%2F+sum%28go_gc_heap_live_bytes%29%29&g1.show_tree=0&g1.tab=table&g1.range_input=1h&g1.res_type=auto&g1.res_density=medium&g1.display_mode=lines&g1.show_exemplars=0). Okay.